Assets/Scripts/Player/PlayerController.cs

```diff
@@ -536,6 +537,7 @@ private void HandleDash()
             }
             else if (PlayerState is PlayerStateEnum.Dash)
             {
+                _shake.Shake(1f,2f,true,true); //TODO: change temp parameters for visual testing
```

Comment:
not sure if it's intentional to create a new camera shake every frame of the dash, but if not, you would probably want to put this in the if block above for starting the dash rather than during the dash